Spices/Oregano.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace Spices;
+
+class Oregano
+{
+    private string $oregano;
+
+    public function __construct(string $oregano)
+    {
+        $this->oregano = $oregano;
```

Comment:
Šis visiem ir viens, kas norāda, ka var izmantot parent klasi un vienotu metodi - "getName".
Nav pabeigts uzdevums, jo nav izmantota mantošana, kas bija galvenais nosacījums.